lib/pages/login.dart

```diff
@@ -11,7 +11,16 @@ class LoginPage extends StatefulWidget {
   _LoginPageState createState() => new _LoginPageState();
 }
 
-final GoogleSignIn googleSignIn = GoogleSignIn();
+final GoogleSignIn googleSignIn = GoogleSignIn(
```

Comment:
```suggestion
final GoogleSignIn googleSignIn = new GoogleSignIn(
```
Try changing this too!

Reply:
@nishantkr18